trie/triedb/pathdb/database.go

```diff
@@ -441,3 +441,26 @@ func (db *Database) Head() common.Hash {
 	defer db.lock.Unlock()
 	return db.tree.front()
 }
+
+// ContainRootHash returns whether MPT root hash is existent.
+func (db *Database) ContainRootHash(root common.Hash) bool {
+	db.lock.Lock()
+	defer db.lock.Unlock()
+
+	bottom := db.tree.bottom()
+	if l := db.tree.get(root); l != nil {
+		if l.rootHash() == bottom.rootHash() {
+			log.Info("root hash is equal to disk layer root")
```

Comment:
Capitalize the first letter of the log to keep a consistent style.

Reply:
It makes more sense to add `root` to the log context.

Reply:
fixed